### DISCUSSION TITLE
侧边栏底部的社交图标是怎么添加的呀

### DISCUSSION THREAD

**Discussion by Achuan-2** [xaoxuu/hexo-theme-stellar#135] [Q&A] [answered]
![Snipaste_2022-10-26_23-55-11](https://user-images.githubusercontent.com/60436214/198075455-68d16819-954d-48d4-9fa8-51bd6ea5609a.png)

**Comment by xaoxuu** [comment, marked as answer]
```yaml
footer:
  social:
```

**Comment by Achuan-2** [comment]
okkk，感谢
